telegram/constants.py

```diff
@@ -149,13 +149,59 @@
     MAX_POLL_QUESTION_LENGTH (:obj:`int`): 300
     MAX_POLL_OPTION_LENGTH (:obj:`int`): 100
 
-:class:`telegram.files.MaskPosition`:
+:class:`telegram.MaskPosition`:
 
 Attributes:
     STICKER_FOREHEAD (:obj:`str`): 'forehead'
     STICKER_EYES (:obj:`str`): 'eyes'
     STICKER_MOUTH (:obj:`str`): 'mouth'
-    STICKER_CHIN (:obj:`str`): 'chin'
+    STICKER_CHIN (:obj:`str`): 'chin'́
```

Comment:
that's a front tick?

```suggestion
    STICKER_CHIN (:obj:`str`): `'chin'́`

Reply:
upsie, my keyboard sometimes does stuff … thanks for the catch!